packages/medusa/src/models/product-option.ts

```diff
@@ -18,7 +18,9 @@ export class ProductOption extends SoftDeletableEntity {
   @Column()
   title: string
 
-  @OneToMany(() => ProductOptionValue, (value) => value.option)
+  @OneToMany(() => ProductOptionValue, (value) => value.option, {
+    cascade: ["soft-remove"],
```

Comment:
suggestion: We should also add 'remove' if we want to remove them permanently. wdyt?

Reply:
I initially added it only for soft-remove since we dont remove the productOptions, but I dont mind adding 'remove' as well

Reply:
it is more a thought than a must, I let you decide that one. My thinking is that is a user want to remove them totally using the repo, there is no cascade and no way for him to customise the entity relation options :) but it was more of a non blocking thought

Reply:
I believe we can safely hard remove both product options and product option values. Can't find a scenario where we would need to keep deleted ones 🤔